packages/basti/src/cli/run.ts

```diff
@@ -45,10 +45,15 @@ void yargs(hideBin(process.argv))
           type: 'string',
           description: 'ID of the RDS cluster to be initialized',
         })
-        .option('elasticache-cluster', {
+        .option('elasticache-redis-cluster', {
```

Comment:
Please, add aliases for existing fields for backward compatibility. There might already be people using the `--elasticache-cluster` and `--elasticache-node` arguments. Here's an example of an alias in from Basti codebase: https://github.com/BohdanPetryshyn/basti/blob/bee65961d3923c10db54dfc4a1318ccbd8b48f59/packages/basti/src/cli/run.ts#L215